changelog/highcharts/9.0.0.md

```diff
@@ -7,6 +7,7 @@
 
 ## Upgrade notes
 - Due to improved security, some tags or attributes may be removed from HTML configuration options like labels or tooltip formats and formatters. Read more [in the docs](https://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting#filtering) on how to allow them.
+- IndicatorRequired mixin no longer works, because the class architecture (related to tree-shaking) has changed.
```

Comment:
1. I'm afraid this might not be really clear to users. Can we simply inform that error of missing a technical indicator dependency requirements will no longer be there? 

2. Wrong file. Not Highcharts changelog, but Stock should be updated: https://github.com/highcharts/highcharts/blob/master/changelog/highcharts-stock/9.0.0.md